extensions/exposition/documentation/authorities.md

```diff
@@ -0,0 +1,53 @@
+# Authorities
+
+Authorities are a mechanism that allows serving multiple domains from a single instance of the
+application.
+
+## Definition
+
+The `authorities` definition is a map of authority identifiers to the `:authority` pseudo-header
+values.
+
+```yaml
+# context.toa.yaml
+
+exposition:
+  authorities:
+    one: the.one.com
+    two: the.two.com
+```
```

Comment:
Why do we need this map?

Reply:
Domain names are transient while authority identifiers are [constant](https://github.com/toa-io/toa/blob/feat/realms/extensions/exposition/documentation/authorities.md#identity).